stack.yaml

```diff
@@ -13,22 +13,22 @@ packages:
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
 extra-deps:
 - concurrent-supply-0.1.8
-- language-lua-0.11.0.1
+- language-lua-0.11.0.2
 - git: https://github.com/jkoppel/language-javascript.git
   commit: 516f52b13870fa7d99b0503a0730438f07891340
-- language-dot-0.1.1
+- language-dot-0.1.2
 - git: https://github.com/jkoppel/language-c.git
-  commit: 2aee906576087ba3d5a1bd7ebc86f30bf8416470
+  commit: 9a496ea8da36742ab3c4ca7769f072f8f6091deb
 - git: https://github.com/jkoppel/language-java.git
```

Comment:
Make sure tutorial and sample app and cubix are on the same versions of _all_ dependencies, including language-java, language-javascript etc.